Makefile

```diff
@@ -615,7 +622,7 @@ realclean: clean java_clean
 	@$(RM) -f *.bin debian/changelog*
 
 ### additionally removes generated files not in the repository tree
-distclean: realclean clean_genRules
+distclean: clean_stack realclean clean_genRules
```

Comment:
Actually distclean should remove everything, what got produced during one or more runs, i.e. rollback in a way that it appears as make has never been run. So I guess, there needs to be done more than just rm stack's work dir?